pages/upgrade/rolling_es_upgrade.rst

```diff
@@ -0,0 +1,32 @@
+.. _es_rolling_upgrade:
+
+***********************************
+Elasticsearch Rolling Upgrade Notes
+***********************************
+
+This page contains a few notes and a recommended procedure to perform a rolling upgrade for a Elasticsearch cluster utilized by Graylog.
+
+Elasticsearch supports rolling upgrades to avoid downtimes during upgrades. Detailed information about the procedures and limitations are provided `here <https://www.elastic.co/guide/en/elasticsearch/reference/master/rolling-upgrades.html>`__.
+
+Graylog supports rolling upgrades without restarting any Graylog node for Elasticsearch upgrades *between minor versions*. While Elasticsearch supports upgrading from e.g. 6.8 to 7.0, Graylog requires a restart when the major version of the Elasticsearch cluster is changed.
+
+To avoid message loss in case of a rolling Elasticsearch upgrade which requires the restart of Graylog nodes, please follow this procedure:
+
+1. Temporarily disable ES automatic node discovery if it is used.
```

Comment:
I'd link this to `https://docs.graylog.org/en/latest/pages/configuration/elasticsearch.html#automatic-node-discovery` so people know which discovery is meant.

Reply:
:+1: